app/src/main/res/values-sw600dp-land/dimens.xml

```diff
@@ -420,4 +420,8 @@
   <dimen name="revision_card_fragment_layout_text_margin_top">32dp</dimen>
   <dimen name="revision_card_fragment_layout_button_margin_top">24dp</dimen>
 
+  <!-- Hints and Solution Fragment -->
+  <dimen name="hints_and_solution_recycler_view_padding_top">4dp</dimen>
+  <dimen name="hints_and_solution_recycler_view_padding_bottom">112dp</dimen>
+  <dimen name="hints_and_solution_toolbar_height">64dp</dimen>
```

Comment:
Where did this `64dp` value came from? In layout files I am unable to see any file which has this 64dp toolbar height.

Reply:
Since I can't use `?attr/actionBarSize` inside dimens.xml, I used the equivalent value in dp for all the 4 layouts files.